### DISCUSSION TITLE
Call to undefined method App\Models\User::hasRole()

### DISCUSSION THREAD

**Discussion by VerlengsnoerNL** [spatie/laravel-permission#1742] [Q&A] [answered]
Call to undefined method App\Models\User::hasRole() (View: /var/www/html/clientarea/resources/views/layouts/header.blade.php)

Wanneer ik @hasrole('admin') in het bestand zet, please help !!

**Comment by drbyte** [comment, marked as answer]
Must add the `HasRoles` trait to your User model, as described in the documentation:
https://spatie.be/docs/laravel-permission/v4/basic-usage/basic-usage
